server/src/http/feature_refresher.rs

```diff
@@ -299,6 +311,97 @@ impl FeatureRefresher {
         }
     }
 
+    /// This is where we set up a listener per token.
+    #[cfg(feature = "streaming")]
+    pub async fn start_streaming_features_background_task(&self) -> anyhow::Result<()> {
```

Comment:
Because this came up as a question during code walkthrough in the spike: I'm using anyhow here to allow us to easily return errors to use the `?` operator. We should probably make it an EdgeResult or something instead, but hey, it's a spike.